npm/src/upload/upload-form.ts

```diff
@@ -405,4 +416,28 @@ export class UploadForm {
       )
     }
   }
+
+  private checkForCorrectJudgmentFileExtension(fileName: string) {
+    const acceptableJudgmentFileExtensions = [
+      ".doc",
+      ".docm",
+      ".docx",
+      ".dot",
+      ".dotm",
+      ".dotx"
+    ]
+
+    if (fileName) {
+      const indexOfLastDot = fileName.lastIndexOf(".")
+      const fileExtension = fileName.slice(indexOfLastDot)
+      if (!acceptableJudgmentFileExtensions.includes(fileExtension)) {
+        this.rejectUserItemSelection(
+          this.warningMessages?.incorrectFileExtensionMessage,
+          "Only MS Word docs are allowed to be selected"
```

Comment:
Should we be consistent with the error messages and use "Microsoft" instead of "MS"? Also not sure whether we should do some user research on the use of "Microsoft" / "MS"? Wonder whether user would understand one better that the other?

Reply:
Ah, this message `Only MS Word docs are allowed to be selected` is the exception message, the actual warning text is in the views. The `rejectUserItemSelection` method takes the CSS id of the warning message (in order to unhide the message) and the exception message.